pkg/server/server.go

```diff
@@ -302,8 +303,12 @@ func (s *Server) HTTPHandler() http.Handler {
 
 	// Helm Endpoints
 	helmConfig := &handlers.HelmHandlers{
-		ApiServerHost: s.KubeAPIServerURL,
-		Transport:     s.K8sClient.Transport,
+		ApiServerHost:           s.KubeAPIServerURL,
```

Comment:
use a constructor, as demonstrated in https://github.com/akashshinde/console/pull/3 - `server` does not need to know anything about `actions` package.